doc/user/ospfd.rst

```diff
@@ -660,6 +660,14 @@ Interfaces
    point-to-point, but the HUB will be a point-to-multipoint. To make this
    topology work, specify the optional 'dmvpn' parameter at the spoke.
 
+   When the network is configured as point-to-multipoint and `delay-reflood`
+   is specified, LSAs received on the interface from neighbors on the
+   interface will not be flooded back out on the interface immediately.
+   Rather, they will be added to the neighbor's link state retransmission
+   list and only sent to the neighbor if not the neighbor doesn't
```

Comment:
This sounds weird to me (or just me :smile: ): `if not the neighbor doesn't`.

Reply:
The intent of the original change was to not have all the OSPF routers attached to the P2MP network reflood when multicast is supported and many of the routers would have already received the latest instance of the LSA. Especially with many wireless P2MP interfaces, the P2MP routers support multicast natively and there are overlapping P2MP meshes of routers with multicast connectivity.  However, there were some P2MP topologies where the prior change is undesirable since it delayed LSDB convergence and this PR reverts to the default behavior of reflooding immediately. With this PR, reflooding back out a P2MP interface is only delayed when explicitly configured. Make sense?